content/articles/pathlib-vs-os-modules-for-file-and-directory-system-operation/index.md

```diff
@@ -0,0 +1,910 @@
+### Pathlib vs OS Modules for File and Directory System Operation
+
+In this tutorial, we will run through different viable contexts to discover an agreeable decision of utilizing either Pathlib or OS for file system operation. Also, we will utilize this tutorial to perform distinguished tasks for significant reference, these tasks are not limited to writing and reading a file, describing a document information, renaming files, creating directories and so on.
+
+### Prerequisites
+To follow this tutorial all it requires is the following:
+
+- Basic knowledge in python programming language.
+
+### Goals
+By the end of this tutorial, the reader should be acquainted with:
+
+- Utilizing pathlib and os modules to manipulating a file and a directory path.
+- Implementing object-oriented programming and functional programming in file system operation.
+- Converting a file's metadata to basic readable information.
+- Manipulating relative path and absolute path.
+- Understanding the scenario where os and pathlib modules can be independently fit in.
+
+### Contents
+- [Introduction](#introduction)
+- [Os module](#os-module)
+- [Pathlib module](#pathlib-module)
+- [Highlighting the practical difference between os and pathlib module](#highlighting-the-practical-difference-between-os-and-pathlib-module)
+- [Querying current path using os and pathlib](#querying-current-path-using-os-and-pathlib)
+- [Navigating to home directory](#navigating-to-home-directory)
+- [Listing a directory path content](#listing-a-directory-path-content)
+- [Creating a new directory and file](#creating-a-new-directory-and-files)
+- [Checking for an existing file or directory](#checking-for-an-existing-file-or-directory)
+- [Renaming a directory and file](#renaming-a-directory-or-file)
+- [Joining path components together](#joining-path-component-together)
+- [Querying path to retrieve a final file or directory name](#querying-path-to-retrieve-a-final-file-or-directory-name)
+- [Removing a directory](#removing-a-directory)
+- [Removing a file](#removing-a-file)
+- [Retrieving and converting a file information](#retrieving-and-converting-a-file-information)
+- [Querying a file extension and name](#querying-a-file-extension-and-name)
+- [Theoretical difference between pathlib and Os](#theoretical-difference-between-pathlib-and-os)
+- [Wrapping Up](#wrapping-up)
+
+### Introduction
+File System Operation can be characterized as the example or technique that can be used to control or construct files and directories and there are two different ways of working with file system operation, which are functional dependency and object-oriented concept. This tutorial practically highlights the different scenarios to differentiate between Pathlib and Os.
```

Comment:
Make sure you maintain consistency while mentioning the name of the libraries - `Pathlib` and `OS`.

Do this change throughout the article